Platform/de.lproj/Localizable.strings

```diff
@@ -898,10 +898,10 @@
 "Timed out waiting for RPC." = "Zeitüberschreitung bei RPC.";
 
 /* VMDisplayAppleWindowController */
-"To access the shared directory, the guest OS must have Virtiofs drivers installed. You can then run `sudo mount -t virtiofs share /path/to/share` to mount to the share path." = "Um auf das freigegebene Verzeichnis zugreifen zu können, müssen auf dem Gastbetriebssystem Virtiofs-Treiber installiert sein. Führen Sie anschließend `sudo mount -t virtiofs share /path/to/share` aus, um den Freigabepfad zu mounten.";
+"To access the shared directory, the guest OS must have Virtiofs drivers installed. You can then run `sudo mount -t virtiofs share /path/to/share` to mount to the share path." = "Um auf das freigegebene Verzeichnis zugreifen zu können, müssen auf dem Gastbetriebssystem VirtIOfs-Treiber installiert sein. Führen Sie anschließend `sudo mount -t virtiofs share /path/to/share` aus, um den Freigabepfad zu mounten.";
 
 /* VMMetalView */
-"To capture input or to release the capture, press Command and Option at the same time." = "Drücken Sie cmd und alt zugleich, um Tastatur und Mauszeiger zu fangen oder wieder freizugeben.";
+"To capture input or to release the capture, press Command and Option at the same time." = "Drücken Sie gleichzeitig CMD+OPT (⌘+⌥), um die Tastatur und den Mauszeiger nur in UTM zu nuzten oder wieder für alle andere Anwendungen freizugeben.";
```

Comment:
〽️ The word "ausschließlich" ("solely") would improve the meaning of the sentence:

```suggestion
"To capture input or to release the capture, press Command and Option at the same time." = "Drücken Sie gleichzeitig CMD+OPT (⌘+⌥), um die Tastatur und den Mauszeiger ausschließlich in UTM zu nuzten oder wieder für alle andere Anwendungen freizugeben.";
```

Reply:
```suggestion
"To capture input or to release the capture, press Command and Option at the same time." = "Drücken Sie gleichzeitig CMD+OPT (⌘+⌥), um die Tastatur und den Mauszeiger ausschließlich in der VM zu nutzen oder wieder für alle andere Anwendungen freizugeben.";
```
Fixed typo and changed UTM to VM.